sliding_window.py

```diff
@@ -0,0 +1,19 @@
+def sliding_window(array, window_size):
```

Comment:
Mi6: can you please add a docstring

Reply:
Working on it...

*This is an automated message generated by Mi6*